src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+let data = document.getElementsByClassName('population');
+
+data = [...data].map(tag =>
```

Comment:
it's better if the type of your variable always the same, just create another variable for this

Reply:
Ok!